integrations/octopus/.port/resources/blueprints.json

```diff
@@ -0,0 +1,292 @@
+[
+  {
+    "identifier": "octopusDeployProject",
+    "title": "Octopus Deploy Project",
+    "icon": "Octopus",
+    "description": "An Octopus Deploy project",
+    "schema": {
+      "properties": {
+        "slug": {
+          "type": "string",
+          "title": "Slug",
+          "description": "The slug identifier of the project"
+        },
+        "id": {
+          "type": "string",
+          "title": "Project ID",
+          "description": "The unique identifier of the project"
+        },

```

Comment:
Let's rely on the entity `identifier` to identify project so there will be no need for this